experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -350,12 +416,16 @@ def test(*options, cfg=None, debug=False):
     running_metrics_overall = runningScore(n_classes)
 
     # Augmentation
-    section_aug = Compose([Normalize(mean=(config.TRAIN.MEAN,), std=(config.TRAIN.STD,), max_pixel_value=1,)])
+    section_aug = Compose(
+        [Normalize(mean=(config.TRAIN.MEAN,), std=(config.TRAIN.STD,), max_pixel_value=1,)]
+    )
 
     patch_aug = Compose(
         [
             Resize(
-                config.TRAIN.AUGMENTATIONS.RESIZE.HEIGHT, config.TRAIN.AUGMENTATIONS.RESIZE.WIDTH, always_apply=True,
+                config.TRAIN.AUGMENTATIONS.RESIZE.HEIGHT,
+                config.TRAIN.AUGMENTATIONS.RESIZE.WIDTH,
+                always_apply=True,
```

Comment:
Are we sure we want to resize the patches before scoring them? I'm not sure  myself, In the quick experiment I ran today on training with and without resizing and padding, the former was better (especially for smaller classes -- one nice outcome of adding class IoUs in Tensorboard)

Reply:
discussed - added TODO for another PR